src/context.rs

```diff
@@ -222,7 +222,7 @@ impl AppContext {
         match self.network {
             Network::Dash => "dash".to_string(),
             Network::Testnet => "testnet".to_string(),
-            Network::Devnet => format!("devnet:{}", self.devnet_name.clone().unwrap_or_default()),
+            Network::Devnet => "devnet".to_string(),
```

Comment:
why change this?

Reply:
It is confusing, usually we just do `network.to_string()` in the code, what leads to bugs.

As discussed, we drop multi-devnet support until it proves useful.